keyboards/trnthsn/s6xty5/keyboard.json

```
@@ -0,0 +1,438 @@
+{
+    "manufacturer": "ThanhSon.Mech",
+    "keyboard_name": "S6xty5",
+    "maintainer": "trnthsn",
+    "bootloader": "stm32-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["B0", "F1", "F0", "C15", "C14", "C13", "A7", "B9", "B8", "B7", "B6", "B5", "B4", "B3", "A15"],
+        "rows": ["B10", "B11", "A2", "A0", "A1"]
+    },
+    "rgblight": {
+        "animations": {
+            "alternating": true,
+            "breathing": true,
+            "christmas": true,
+            "knight": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "rgb_test": true,
+            "snake": true,
+            "static_gradient": true,
+            "twinkle": true
+        },
+        "led_count": 22,
+        "max_brightness": 240,
+        "sleep": true
+    },
+    "ws2812": {
+        "pin": "B15",
+        "driver": "spi"
+    },
+    "indicators": {
+        "caps_lock": "A3",
+        "on_state": 0
+    },
+    "processor": "STM32F072",
+    "url": "",
+    "usb": {
+        "vid": "0x5453",
+        "pid": "0x5335",
+        "device_version": "0.0.1"
+    },
+    "layout_aliases": {
+        "LAYOUT": "LAYOUT_all"
+    },
+    "community_layouts": [
+        "65_ansi_blocker",
+        "65_ansi_blocker_split_bs",
+        "65_ansi_blocker_tsangan",
+        "65_ansi_blocker_tsangan_split_bs"
+    ],
```

1. layout aliases not required
2. community layout one lined
```suggestion
    "community_layouts": ["65_ansi_blocker", "65_ansi_blocker_split_bs", "65_ansi_blocker_tsangan", "65_ansi_blocker_tsangan_split_bs"],
```

---

I updated